### PR TITLE
Shayan/adding feature of active market to chart title

### PR DESCRIPTION
**Pull request by shayan-binary-2**

### CODE REVIEW

File: app/test.jsx [hunk 1]
```diff
@@ -452,10 +453,23 @@ class App extends Component {
         window.location.href = `${baseUrl}?marketsOrder=${evt.target.value}`;
     };
 
-    onActiveCategory = () => {
-        this.setState(prevState => ({
-            activeCategory: prevState.activeCategory ? null : 'synthetic_index',
-        }));
+    onActiveMarket = (evt) => {
+        const marketAray = evt.target.value.split(',');
```
Comment (balakrishna-binary) [inline app/test.jsx:457]:
```suggestion
        const marketArray = evt.target.value.split(',');
```
typo

File: app/test.jsx [hunk 2]
```diff
@@ -251,9 +251,10 @@ class App extends Component {
             <ChartTitle
                 onChange={this.symbolChange}
                 active_category={this.state.activeCategory}
```
Comment (balakrishna-binary) [inline app/test.jsx:253]: should we remove this line since we don't use `activeCategory` from state anymore?

File: src/components/ChartTitle.jsx [hunk 3]
```diff
@@ -9,6 +9,7 @@ class ChartTitle extends Component {
         const { updateProps: prevUpdateProps, ...previousProps } = prevProps;
         if (
             (previousProps.active_category !== props.active_category)
```
Comment (balakrishna-binary) [inline src/components/ChartTitle.jsx:11]: do we still need this check?

File: app/test.jsx (hunk 1 above)
Comment (shayan-binary-2) [reply app/test.jsx]: @balakrishna-binary just fixed it

File: app/test.jsx (hunk 2 above)
Comment (shayan-binary-2) [reply app/test.jsx]: @balakrishna-binary just fixed it

File: src/components/ChartTitle.jsx (hunk 3 above)
Comment (shayan-binary-2) [reply src/components/ChartTitle.jsx]: @balakrishna-binary just fixed it

**Review by balakrishna-binary — APPROVED** [review]

**Comment by shayan-binary-2** [comment]
Rename the prop and naming, `open_market` would be more meaningfull

**Review by zaki-binary — APPROVED** [review]

**Pull request merged**